File: python/hello.py
# number = 8
# print("请输入数字：")
# temp = int(input("数字："))
# while (number != temp):
#     if (temp < number):
#         print("小了")
#     else:
#         print("大了")
#     print("请输入数字：")
#     temp = int(input("数字："))

# print("恭喜猜对了")

# str = 'hia'
# print(str * 3)  # *可以复制
# print('h' in str)  # in可以判断是否在字符串里面，结果是boolean类型的值

# arr = ['1q','2c','3d','4f']
# print('3c' in arr)

#函数：
'''
def 函数名():
    代码块...

函数的调用：

函数名()
'''


def info(height, weight):
    print('小张的身高是%f' % height)
    print('小张的体重是%f' % weight)
    pass


info(1.68, 120)
# *args 表示可变长度的参数，接受的是元祖类型


def getSum(*args):
    sum = 0
    for item in args:
        sum += item
        pass
    return sum
    pass


print(getSum(1, 2, 3))


# 关键字可变参数：接受的是一个字典类型
def keyFunc(**kwArgs):
    print(kwArgs)
    pass


keyFunc(**{'name': 'Eva', 'age': 18})

File: lnj/ts/js/hello.js
"use strict";
//由于TS并不是一门新的语言，而是对JS的扩展，所以可以在TS文件中直接编写JS代码
//TS支持类型注解，我们可以通过类型注解来告诉系统，变量中将来只能存储什么类型的数据
let val;
val = 999;
function sum(a, b) {
    return a.length + b;
}
console.log(sum([1, 2, 3], 99));
let flag;
flag = true;
console.log(flag, "flag");
let name1;
name1 = "Eva";
console.log(name, "name");
